examples/vision/image-classification/pets/quantize_vit_model.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import time
+from tempfile import NamedTemporaryFile
+
+import torch
+import torch.nn.functional as F
+from accelerate import init_empty_weights
+from datasets import load_dataset
+from safetensors.torch import load_file, save_file
+from transformers import (
+    ViTForImageClassification,
+    ViTImageProcessor,
+    ViTConfig,
+)
+
+from optimum.quanto import (
+    Calibration,
+    QTensor,
+    freeze,
+    qfloat8,
+    qint4,
+    qint8,
+    quantization_map,
+    quantize,
+    requantize,
+)
+
+
+def test(model, device, test_loader):
+    model.to(device)
+    model.eval()
+    test_loss = 0
+    correct = 0
+    with torch.no_grad():
+        start = time.time()
+        for batch in test_loader:
+            data, target = batch["pixel_values"], batch["labels"]
+            data, target = data.to(device), target.to(device)
+            output = model(data).logits
+            # print("*****I am after output", output)
```

Comment:
```suggestion
```

Reply:
removed it in https://github.com/huggingface/optimum-quanto/pull/308/commits/bd0332fbaac1d7f580a1be7cadb0c972f6779536